Readme.md

```diff
@@ -76,7 +77,7 @@ See [slurm.conf](https://slurm.schedmd.com/slurm.conf.html#OPT_PlugStackConfig)
 [spank](https://slurm.schedmd.com/spank.html#SECTION_CONFIGURATION) documentation for details.
 
 
-## Redistribution
+## TODO Redistribution
 
 Build an RPM source package using `make rpm`. The resulting RPM is located in the current directory
```

Comment:
Should this be replaced with a reference to building the RPM using `rpm/prepare-rpm.sh`?

Reply:
Thanks, yes, I forgot it. I've updated the readme and renamed the script to `make-rpm.sh`.